packages/scene-composer/src/components/panels/scene-components/tag-style/ColorPicker/ColorPickerUtils/ColorPickerStyles.ts

```diff
@@ -0,0 +1,65 @@
+import { CSSProperties } from 'styled-components';
+
```

Comment:
the colors in this file should from design tokens instead of hard coded strings so it can support light/dark mode easily

Reply:
Done